src/concrete/ml/sklearn/linear_model.py

```diff
@@ -835,61 +819,24 @@ def partial_fit(
             # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4184
             raise NotImplementedError("Partial fit is not currently supported for clear training.")
 
-    # This method is taken directly from scikit-learn
-    def _predict_proba_lr(self, X: Data, fhe: Union[FheMode, str]) -> numpy.ndarray:
-        """Probability estimation for OvR logistic regression.
-
-        Positive class probabilities are computed as
-        1. / (1. + np.exp(-self.decision_function(X)));
-        multiclass is handled by normalizing that over all classes.
-
-        Args:
-            X (Data): The input values to predict, as a Numpy array, Torch tensor, Pandas DataFrame
-                or List. It mush have a shape of (n_samples, n_features).
-            fhe (Union[FheMode, str]): The mode to use for prediction.
-                Can be FheMode.DISABLE for Concrete ML Python inference,
-                FheMode.SIMULATE for FHE simulation and FheMode.EXECUTE for actual FHE execution.
-                Can also be the string representation of any of these values.
-
-        Returns:
-            numpy.ndarray: The predicted class probabilities.
-        """
-        prob = self.decision_function(X, fhe=fhe)
-        prob = numpy_sigmoid(prob)[0]
-
-        assert isinstance(prob, numpy.ndarray)
-
-        if prob.shape[1] == 1:
-            prob = prob.flatten()
-            return numpy.vstack([1 - prob, prob]).T
-
-        # OvR normalization, like LibLinear's predict_probability
-        prob /= prob.sum(axis=1).reshape((prob.shape[0], -1))
-
-        return prob
-
-    def predict_proba(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
-        """Probability estimates.
+    def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
```

Comment:
basically we don't need to define the `predict_proba` method as it is in sklearn, we only need to re-define `post_processing` with sklearn's implem